src/sentry/static/sentry/app/components/group/externalIssueForm.jsx

```diff
@@ -0,0 +1,253 @@
+import $ from 'jquery';
+import React from 'react';
+import PropTypes from 'prop-types';
+import queryString from 'query-string';
+import {debounce} from 'lodash';
+
+import {addSuccessMessage} from 'app/actionCreators/indicator';
+import AsyncComponent from 'app/components/asyncComponent';
+import FieldFromConfig from 'app/views/settings/components/forms/fieldFromConfig';
+import Form from 'app/views/settings/components/forms/form';
+import SentryTypes from 'app/sentryTypes';
+import {t} from 'app/locale';
+import ExternalIssueStore from 'app/stores/externalIssueStore';
+
+const MESSAGES_BY_ACTION = {
+  link: t('Successfully linked issue.'),
+  create: t('Successfully created issue.'),
+};
+
+const SUBMIT_LABEL_BY_ACTION = {
+  link: t('Link Issue'),
+  create: t('Create Issue'),
+};
+
+class ExternalIssueForm extends AsyncComponent {
+  static propTypes = {
+    group: SentryTypes.Group.isRequired,
+    integration: PropTypes.object.isRequired,
+    action: PropTypes.oneOf(['link', 'create']),
+    onSubmitSuccess: PropTypes.func.isRequired,
+  };
+
+  shouldRenderBadRequests = true;
+
+  getEndpoints() {
+    const {integration} = this.props;
+    return [['integrationDetails', integration.url]];
```

Comment:
I'm not sure where `integration.url` is coming from but this failed for me so I'd put back
```
return [
      [
        'integrationDetails',
        `/groups/${group.id}/integrations/${integration.id}/?action=${action}`,
      ],
]
```